examples/mnist-ddp/gpu/variables.bash

```diff
@@ -0,0 +1,7 @@
+APP_NAME=mypytorchapp
+
+KF_ENV=jinanpytorchenv
+
+NAMESPACE=jinanpytorch
```

Comment:
Is this file used in the example?

Reply:
Yes. variables.bash will be sourced when running install.bash